app/services/store_participant_eligibility.rb

```diff
@@ -24,6 +18,17 @@ def call
 
 private
 
+  attr_reader :participant_profile, :eligibility_options
+
+  def initialize(participant_profile:, eligibility_options: {})
+    @participant_profile = participant_profile
+    @eligibility_options = eligibility_options
+  end
+
+  def eligibility_triggers!
+    RecordDeclarations::Actions::MakeDeclarationsEligibleForParticipantProfile.call(participant_profile: participant_profile)
+  end
```

Comment:
Do we need to be able to go the other way? i.e when participant becomes ineligible, change all eligible declarations to submitted?